src/bb/contract.rs

```diff
@@ -26,7 +25,7 @@ impl ContractCommand {
             .arg("-k")
             .arg(self.path_to_vk)
             .arg("-o")
-            .arg(self.path_to_contract_output);
+            .arg("/dev/null");
```

Comment:
Note: windows does not have /dev/null !

Reply:
Windows has `NUL` however which we can use here.